ee/clickhouse/queries/person_query.py

```diff
@@ -1,36 +1,49 @@
-from typing import Optional, Union
+from typing import List, Optional, Union
 
+from ee.clickhouse.materialized_columns.columns import ColumnName
 from ee.clickhouse.queries.column_optimizer import ColumnOptimizer
 from posthog.models import Filter
 from posthog.models.filters.path_filter import PathFilter
 
 
 class ClickhousePersonQuery:
     PERSON_PROPERTIES_ALIAS = "person_props"
```

Comment:
Delete this now and just use the dict in the future?

Reply:
Other places rely on this externally